### DISCUSSION TITLE
Offset Not Working

### DISCUSSION THREAD

**Discussion by Arinji2** [pocketbase/pocketbase#3971] [Q&A]
I am just confused, how do i have an offset lol

https://db-news.arinji.com/api/collections/live/records?page=1&perPage=5&offset=10

there is a total of 9 results, why is the offset not working

**Comment by ganigeorgiev** [comment]
PocketBase doesn't have `offset` query parameter.

The `page` and `perPage` params are used for navigating over the paginated results list.

**Reply by ganigeorgiev** [reply]
You can explore all available list query parameters under the "Query parameters" section in https://pocketbase.io/docs/api-records/#listsearch-records.

**Reply by Arinji2** [reply]
Ah got it, so how would you recommend i go about this?

**Reply by ganigeorgiev** [reply]
I don't understand what you are trying to do. Again, if you want to navigate over the paginated list use the `page` and `perPage` params.

**Reply by Arinji2** [reply]
Hmm, no worries..thanks :D 
lemme try and make something for this.
